### PR TITLE
feat: Move Angular to HOLD

### PR DESCRIPTION
**Pull request by sclausson**

### CODE REVIEW

File: radar/dev/angular.yaml [hunk 1]
```diff
@@ -3,13 +3,20 @@ logo: https://angular.io/assets/images/logos/angular/angular.png
...
 rationale: |
   Angular is a complete web application framework running on TypeScript. With
   a convention over configuration approach and TypeScript as a requirement it allows
   for an easier transition from backend to frontend and with a complete set
   of tooling it enables us to build large scale web applications.
+
+  We currently have both React and Angular in ADOPT, but we do not intend to support both
+  moving forward. Rallying around a singular frontend technology, we are better able
+  to share knowledge, leverage reusable components, and support a consistent look and flow across
+  the multitude of HII Retail Service web frontends that we will be developing.
```
Comment (Tasseri) [inline radar/dev/angular.yaml:19]: Should it be Hii Retail?

File: radar/dev/angular.yaml [hunk 2]
```diff
@@ -3,13 +3,20 @@ logo: https://angular.io/assets/images/logos/angular/angular.png
...
+    ring: HOLD
 description: |
   Angular is a platform for building mobile and desktop web applications.
 rationale: |
   Angular is a complete web application framework running on TypeScript. With
   a convention over configuration approach and TypeScript as a requirement it allows
   for an easier transition from backend to frontend and with a complete set
   of tooling it enables us to build large scale web applications.
+
+  We currently have both React and Angular in ADOPT, but we do not intend to support both
```
Comment (sasjo) [inline radar/dev/angular.yaml:16]: Can we change the tense here to something like "Formerly, we had both React and Angluar in Adopt ..."? That would feel more natural when this blip has been moved to hold.

**Review by sasjo — COMMENTED** [review]
The rationale is good, but see my comment on tense.

File: radar/dev/angular.yaml (hunk 2 above)
Comment (sclausson) [reply radar/dev/angular.yaml]: done

File: radar/dev/angular.yaml (hunk 1 above)
Comment (sclausson) [reply radar/dev/angular.yaml]: done

**Review by sasjo — APPROVED** [review]

**Pull request merged**